### PR TITLE
implement project config settings screen

### LINKED ISSUE #22: Project Config / Pre-invite
## Project Config / Pre-invite
- User was on the [Project Invite screen](https://github.com/digidem/mapeo-desktop-shell/issues/21) and landed on the project config screen
- User had not invited devices to this project before, so only display the coordinator device
- Only the `Join Project` and `Invite` are clickable
  

Screen Shot 2023-04-10 at 1 53 14 PM

## Todo

- [ ] Build screen as designed 
- [ ] Build menu as designed
- [ ] Display  the `Join Project` which will redirect to https://github.com/digidem/mapeo-desktop-shell/issues/17
- [ ] Display the `Invite` button which will redirect to https://github.com/digidem/mapeo-desktop-shell/issues/25

## Design
Full flow here : https://www.figma.com/file/uzynHXR0xgJu19lsCoJyXN/%F0%9F%90%9A-Shell-App?node-id=4-623&t=jCsvYtfaEBof4KnS-0

### PR DESCRIPTION
**Pull request by achou11**
Closes #22 

Notes:

- The pre-existing options for shared components were not a good fit for the UI that needed implementing here, so I created some seemingly redundant components but kept their location and usage isolated to this UI.

- There are some visual discrepancies between what's implemented here and the mockup (most notably buttons). I noticed many opportunities to get tangled with variants due to inconsistencies, so I opted to lean towards consistency and ease of implementation for now.

- ~For the sake of testing, you'll need to update the `Router` to show the relevant view component in the index path~

---

Preview:

image

### CODE REVIEW

File: src/renderer/src/theme/index.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const OFF_BLACK = '#333333'
 export const MIDNIGHT_BLUE = '#000033'
 export const MID_BLUE = '#19337F'
 export const WARNING_RED = '#D92222'
-export const GREY_LIGHT = '#EEEEEE'
+export const GREY_LIGHT = '#E6E6E6'
```
Comment (achou11) [inline src/renderer/src/theme/index.ts:12]: seems that this is only being used in screens relevant to this PR, so figured it should be safe to change

File: src/renderer/src/views/Home/Settings/ProjectConfig/ManageTeamSection.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          <Text size="small" fontWeight="600">
+            {member.name}
+          </Text>
+          <Text size="small" color={theme.grey.main}>
+            {new Date(member.lastSynced).toLocaleDateString()}
+          </Text>
+        </Row>
+        <Row justifyContent="space-between">
+          <Text size="small" color={theme.grey.main}>
+            {member.isSelf ? 'This Device!' : member.deviceId}
```
Comment (achou11) [inline src/renderer/src/views/Home/Settings/ProjectConfig/ManageTeamSection.tsx]: does `This Device!` need to be a translated string? Couldn't tell if that was an explicit design choice or just some placeholder text in the design

File: src/renderer/src/views/Home/index.tsx [hunk 3]
```diff
@@ -40,23 +41,25 @@ export const Home = ({ showBottomBar }: Props) => {
   const [activePanel, setPanelName] = React.useState<PanelName>('territory')
 
   return (
-    <DefaultLayout sx={{ display: 'flex', position: 'relative' }}>
```
Comment (achou11) [inline src/renderer/src/views/Home/index.tsx:43]: seems like the language switcher isn't necessary in this view, so decided to not use this layout here

Comment (lightlii) [reply src/renderer/src/views/Home/index.tsx:43]: Slightly off-topic but I'm wondering whether it makes sense to have the lang selector on all pages or just on the navigation view. Will ping Sabella to get her opinion

**Comment by lightlii** [comment]
The navigator view stuff is merged now @achou11 so if you want to rebase we won't have to play around in the router to see this

**Review by ErikSin — APPROVED** [review]

File: src/renderer/src/views/Home/Settings/ProjectConfig/ManageTeamSection.tsx (hunk 2 above)
Comment (ErikSin) [reply src/renderer/src/views/Home/Settings/ProjectConfig/ManageTeamSection.tsx]: lets translate it OR take it out. Just so we don't forget it

Comment (achou11) [reply src/renderer/src/views/Home/Settings/ProjectConfig/ManageTeamSection.tsx]: addressed via https://github.com/digidem/mapeo-desktop-shell/pull/41/commits/efd0dbe1d5dffeac00f0de31158104c581152901 and https://github.com/digidem/mapeo-desktop-shell/pull/41/commits/1b91ba238aa14126117117e525600e138a26a444

**Pull request merged**